warehouse/rss/views.py

```diff
@@ -18,6 +20,19 @@
 from warehouse.xml import XML_CSP
 
 
+def _format_author(release):
```

Comment:
Could you add a quick docstring on this function to explain the rationale behind it?

Reply:
Done.